lib/order.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'texting'
+
+class Order
+
+  include Texting
+
+  attr_reader :basket
+
+  def initialize
+    @basket = []
+  end
+
+  def add_to_basket(menu, dish_number, quantity)
+    quantity.times do
+      @basket << menu.select(dish_number)
+    end
+  end
+
+  def summary
+    fail "Basket empty" if empty_basket?
+    basket_hash = dishes_and_quantities
+    basket_hash.each_pair do |k, v|
+      puts "#{v}x #{k.name} | £#{'%.2f' % (v * k.price)}"
```

Comment:
Style/FormatString: Favor `format` over `String#%`.<br>Rails/Output: Do not write to stdout. Use Rails' logger if you want to log.